app/javascript/components/search/controls/FiltersBoxSearchable.js

```diff
@@ -25,8 +25,16 @@ export default function FiltersBoxSearchable({ facet, selection, setSelection, s
    * Form of IDs: <general name> <specific name(s)>
    * General: All lowercase, specified in app code (e.g. 'apply-facet')
    * Specific: Cased as specified in API (e.g. 'species', 'NCBItaxon9606')
-   *
+   * /single_cell/studies/5e9e07bd771a5b2caa140971/upload
    * UI code concatenates names in the ID.  Names in ID are hyphen-delimited.
+
```

Comment:
Similarly, this specific example looks helpful, but more context in the comment would clarify.